test/mrmatrix_test.py

```diff
@@ -0,0 +1,132 @@
+import unittest
+
+import numpy as np
+from numpy.testing import assert_array_equal
+
+from clodius.tiles.mrmatrix import tileset_info, tiles
+
+class AttrDict(dict):
```

Comment:
What's the point of this class?

Reply:
To allow arbitrary attributes on a dict:
```python
>>> plain_dict = {}
>>> plain_dict.some_attr = 'foo'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'dict' object has no attribute 'some_attr'
>>> class AttrDict(dict): pass
... 
>>> attr_dict = AttrDict()
>>> attr_dict.some_attr = 'foo'
>>> attr_dict.some_attr
'foo'
```

The point is to just to be able to mock up something that looks like an hdf5 object: It has `[]` like a dict, but I also want to add `.attrs`.

Reply:
Maybe calling it `MockHdf5` would make the intention more clear?

Reply:
It definitely would.